components/service/experiments/src/test/java/mozilla/components/service/experiments/KintoExperimentSourceTest.kt

```diff
@@ -40,18 +45,51 @@ class KintoExperimentSourceTest {
         val httpClient = mock(Client::class.java)
 
         val url = "$baseUrl/buckets/$bucketName/collections/$collectionName/records"
+        val json = """
+            {
+                "data":[
+                    {
+                        "id": "first-id",
+                        "id": "first-id",
+                        "description": "first-description",
+                        "last_modified": 1523549895713,
+                        "match": {
+                            "locale_language": "eng",
+                            "app_id": "first-appId",
+                            "regions": []
+                        },
+                        "schema": 1523549592861,
+                        "buckets": {
+                            "start": "0",
+                            "count": "100"
+                        },
+                        "branches": [
+                            {
+                                "name": "branch1",
+                                "ratio": "1"
+                            }
+                        ]
+                    }
+                ]
+            }
+        """.trimIndent()
+
         `when`(httpClient.fetch(any())).thenReturn(
             Response(url,
                 200,
                 MutableHeaders(),
-                Response.Body("""{"data":[{"name":"first-name","match":{"lang":"eng","appId":"first-appId",regions:[]},"schema":1523549592861,"buckets":{"max":"100","min":"0"},"description":"first-description", "id":"first-id","last_modified":1523549895713}]}""".byteInputStream())))
-
-        val expectedExperiment = Experiment("first-id",
-            "first-name",
-            "first-description",
-            Experiment.Matcher("eng", "first-appId", listOf()),
-            Experiment.Bucket(100, 0),
-            1523549895713
+                Response.Body(json.byteInputStream())))
+
+        val expectedExperiment = testDefaultExperiment(
+            id = "first-id",
+            description = "first-description",
+            match = testDefaultMatcher(
+                localeLanguage = "eng",
+                appId = "first-appId",
+                regions = listOf()
+            ),
+            buckets = Experiment.Buckets(0, 100),
+            lastModified = 1523549895713
```

Comment:
nit:  magic number (is knowing the date or this specific date significant to the test?)